templates/_templating_scripting.py

```diff
@@ -21,12 +23,21 @@
 # ensure any new bots have both a "app/" prefix and a "[bot]" postfix version
 BOTS = ["dependabot[bot]", "app/dependabot", "pre-commit-ci[bot]", "app/pre-commit-ci"]
 
+_MAGIC_PREFIX = "@scitools-templating: please"
+MAGIC_NO_PROMPT = re.compile(rf"{_MAGIC_PREFIX} no share prompt")
+MAGIC_NO_NOTIFY = re.compile(rf"{_MAGIC_PREFIX} no update notification on: (\w+)")
```

Comment:
This regex doesn't match hyphenated words, such as `iris-grid`.
The `\w` token is shorthand for `[a-zA-Z0-9_]`; probably because hyphen delimits two words.

Easy fix:
```suggestion
MAGIC_NO_NOTIFY = re.compile(rf"{_MAGIC_PREFIX} no update notification on: ([\w-]+)")
```

Reply:
Also - are there any potential use cases where we might want to specify more than one repo?
I think not, but just putting it out there...